src/pages/SignupPage/SignupPage.tsx

```diff
@@ -0,0 +1,125 @@
+import { useMutation } from '@apollo/client';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import { LoadingButton } from '@mui/lab';
+import {
+  Button,
+  Grid,
+  InputAdornment,
+  TextField,
+  Typography
+} from '@mui/material';
+import React, { FC, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { NavLink, useNavigate } from 'react-router-dom';
+
+import { authService } from '../../graphql/auth/authService';
+import { ISignupResult } from '../../graphql/auth/IAuthResult';
+import { IFormInput } from '../../graphql/auth/IFormInput';
+import { SIGNUP } from '../../graphql/auth/mutation';
+import { schema } from '../LoginPage/validationSchema';
+import { FormAuth, PaperAuth } from './Signup.styles';
+
+const SignupPage: FC = () => {
+  const [signup, { loading }] = useMutation<ISignupResult>(SIGNUP);
+  const navigate = useNavigate();
+  const [hiddenPassword, setHiddenPassword] = useState<boolean>(true);
+  const showPassword = () => {
+    setHiddenPassword((el) => !el);
+  };
```

Comment:
Maybe it's better to separate it by blank lines for better visibility

Reply:
fixed